Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/PingServiceImpl.java

```diff
@@ -58,17 +58,19 @@
     private static final String WSDL_SUFFIX = "?wsdl";
     private static final String LOG_WSDL_KEY = "logWsdlPing";
     private static final String MSG_EXCEPTION_TIMEOUT = "Connection timed out";
+    public static final boolean DO_NOT_IGNORE_DEADHOST = false;
```

Comment:
No need for DO_NOT_IGNORE_DEADHOST. This can be simply negated with IGNORE_DEADHOST